extensions/bond/sawtooth_bond/updates/libor.py

```diff
@@ -198,8 +217,9 @@ def check_valid(self, store, txn):
         #    that provides the LIBORs.
         if self.__libor_object__.Signature is None:
             raise InvalidTransactionError('LIBOR data has not been signed')
-        if not self.__libor_object__.verify_signature(
-                self.__LIBOR_PUBLISHER_ADDR__):
+#         if not self.__libor_object__.verify_signature(
+#                 self.__LIBOR_PUBLISHER_ADDR__):
+        if not self.__libor_object__.verify_signature():
```

Comment:
Does this now mean that any validator can sign LIBOR data?  Originally, the intent was that only LIBOR data from a "trusted oracle" would be accepted.